lib/suse/connect/zypper.rb

```diff
@@ -84,10 +84,11 @@ def repositories
           xml_doc.elements.to_a('stream/repo-list/repo').map { |r| r.to_hash.merge!(url: r.elements['url'].text) }
         end
 
-        # @param service_url [String] url to appropriate repomd.xml to be fed to zypper
+        # @param service_url [String] url to appropriate repoindex.xml to be fed to zypper
```

Comment:
Good catch, see our route in SCC: https://github.com/SUSE/happy-customer/blob/2f4289acf2bd434fc8803fc6bfba73a6c3b7e3b3/glue/config/routes/access.rb